W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/design-api-specification-from-scratch.adoc

With one resource created, you can add a method to it using autocomplete:

> . After `/flights`, press Return to go to a new line and `tab` to indent.

Author: ```suggestion
. On a new line after `/flights`, press Tab to indent.
```

> . Press Ctrl+Spacebar to see all the available options.

Author: ```suggestion
. Press Ctrl+Spacebar to display all the available options.
```

Add specific query parameters for your resources:

. In a new line after the `get:` method of your `/flights` resource, use `tab` to indent.

> . Press Ctrl+Spacebar to see all the available options.

Author: ```suggestion
. Press Ctrl+Spacebar to display all the available options.
```

. Select `queryParameters`.

> . Press Return and then Tab to add a new indentation.

Author: ```suggestion
. Press Return followed by Tab to add a new indentation.
```

> . Press Return followed by Tab to add a new indentation.

Author: ```suggestion
. On a new line, press Tab to add a new indentation.
```
